docs/resources/custom_location.md

```diff
@@ -0,0 +1,46 @@
+---
+page_title: "cpln_custom_location Resource - terraform-provider-cpln"
+subcategory: "Custom Location"
+description: |-
+---
+
+# cpln_custom_location (Resource)
+
+Manages an org's [Custom Location](https://docs.controlplane.com/reference/location#byok-locations).
+
+## Declaration
+
+### Required
+
+- **name** (String) Name of the Custom Location.
+- **tags** (Map of String) Key-value map of resource tags.
+- **enabled** (Boolean) Indication if the custom location is enabled.
+- **provider** (String) Provider of the custom location, must be `byok`.
+
+## Outputs
+
+- **cpln_id** (String) The ID, in GUID format, of the custom location.
+- **description** (String) Description of the custom location.
+- **cloud_provider** (String) Cloud Provider of the custom location (`byok`).
+- **self_link** (String) Full link to this resource. Can be referenced by other resources.
+
+<a id="nestedblock--geo"></a>
+
+## Example Usage
+
+### Hard Code Location Tags
+
+```terraform
+resource "cpln_custom_location" "reference-tags-example" {
```

Comment:
Change the name of the resource, it shouldn't say "reference-tags-example", it should say "example".